src/patternfly/utilities/Text/examples/Text.md

```diff
@@ -66,6 +64,81 @@ import './Text.css'
 {{#> text text--modifier="pf-u-color-400"}}
   Font color 400
 {{/text}}
+<!-- {{#> text text--modifier="pf-u-color-link"}}
+  Link color
+{{/text}} -->
+```
+### Inverse colors
+```hbs
+<div class="pf-u-background-color-dark-400">
+  {{#> text text--modifier="pf-u-color-light-100"}}
+    Font color light 100
+  {{/text}}
+  {{#> text text--modifier="pf-u-color-light-200"}}
+    Font color light 200
+  {{/text}}
+</div>
+```
+### State/status text colors
+```hbs
+{{#> text text--modifier="pf-u-default-color-100"}}
+  Font color default 100
+{{/text}}
+{{#> text text--modifier="pf-u-default-color-200"}}
+  Font color default 200
+{{/text}}
+{{#> text text--modifier="pf-u-default-color-300"}}
+  Font color default 300
+{{/text}}
+{{#> text text--modifier="pf-u-success-color-100"}}
+  Font color success 100
+{{/text}}
+{{#> text text--modifier="pf-u-success-color-200"}}
+  Font color success 200
+{{/text}}
+{{#> text text--modifier="pf-u-info-color-100"}}
+  Font color info 100
+{{/text}}
+{{#> text text--modifier="pf-u-info-color-200"}}
+  Font color info 200
+{{/text}}
+{{#> text text--modifier="pf-u-warning-color-100"}}
+  Font color warning 100
+{{/text}}
+{{#> text text--modifier="pf-u-warning-color-200"}}
+  Font color warning 200
+{{/text}}
+{{#> text text--modifier="pf-u-danger-color-100"}}
+  Font color danger 100
+{{/text}}
+{{#> text text--modifier="pf-u-danger-color-200"}}
+  Font color danger 200
+{{/text}}
+{{#> text text--modifier="pf-u-danger-color-300"}}
+  Font color danger 300
+{{/text}}
+```
+### Disabled text colors
+```hbs
+{{#> text text--modifier="pf-u-disabled-color-100"}}
+  Font color disabled 100
+{{/text}}
+{{#> text text--modifier="pf-u-disabled-color-200"}}
+  Font color disabled 200
+{{/text}}
+{{#> text text--modifier="pf-u-disabled-color-300"}}
+  Font color disabled 300
```

Comment:
Question of where this belongs. This global var is used as a background-color with one exception as a border-color for tertiary nav (that we're deprecating). @mceledonia do you know the intent of this color and where it should belong?